lib/matplotlib/backends/backend_pgf.py

```diff
@@ -623,9 +637,16 @@ def draw_image(self, gc, x, y, im):
         # reference the image in the pgf picture
         writeln(self.fh, r"\begin{pgfscope}")
         self._print_pgf_clip(gc)
-        h, w = im.shape[:2]
         f = 1. / self.dpi  # from display coords to inch
-        writeln(self.fh, r"\pgftext[at=\pgfqpoint{%fin}{%fin},left,bottom]{\pgfimage[interpolate=true,width=%fin,height=%fin]{%s}}" % (x * f, y * f, w * f, h * f, fname_img))
+        if transform is None:
+            writeln(self.fh, r"\pgfsys@transformshift{%fin}{%fin}" % (x * f, y * f))
+            w, h = w * f, h * f
+        else:
+            tr1, tr2, tr3, tr4, tr5, tr6 = transform.frozen().to_values()
+            writeln(self.fh, r"\pgfsys@transformcm{%f}{%f}{%f}{%f}{%fin}{%fin}" % (tr1 * f, tr2 * f, tr3 * f, tr4 * f, (tr5 + x) * f, (tr6 + y) * f))
+            w = h = 1  # scale is already included in the transform
+        interp = str(transform is None).lower()  # interpolation in PDF reader
+        writeln(self.fh, r"\pgftext[left,bottom]{\pgfimage[interpolate=%s,width=%fin,height=%fin]{%s}}" % (interp, w, h, fname_img))
```

Comment:
Okay, modified this PR to fix long lines in my code. No PEP8 errors remaining in the parts I've touched.